tests/integration/test_hermes_release_manifest.py

```
@@ -0,0 +1,116 @@
+
+    assert release_payload["runtime"] == "hermes"
+    assert release_payload["source"]["type"] == "git"
+    assert release_payload["source"]["fork_repo"]
+    assert release_payload["source"]["upstream_repo"]
+    assert release_payload["source"]["pinned_ref"]
+
+    installer = REPO_ROOT / "integrations" / "hermes" / "install.sh"
+    server, authority_url = _start_authority_server()
+    try:
+        env = _base_env(tmp_path, authority_url)
+        result = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)